doc/source/changes/version_0_28.rst.inc

```diff
@@ -44,4 +44,13 @@ Miscellaneous improvements
 Fixes
 -----
 
-* fixed something (closes :issue:`1`).
\ No newline at end of file
+* fixed `Axis.union`, `Axis.intersection` and `Axis.difference` when passed value is a single string
+  (closes :issue:`489`):
+
+    >>> a = Axis('a=a0..a2')
+    >>> a.union('a1..a3')
```

Comment:
The examples here are a bit misleading. One could think the problem was only when using .., when it was the case for "normal" strings too. Does not matter much though as nobody is reading the release notes :(

Reply:
RTFCL (Read The F** Change logs)

Reply:
😁